### PR TITLE
Update css

### PR DESCRIPTION
**Pull request by Luke1982**

### CODE REVIEW

File: Smarty/templates/DetailViewUI.tpl [hunk 1]
```diff
@@ -95,9 +95,9 @@
                     							   <select id="txtbox_{$keyfldname}" name="{$keyfldname}" class="small" style="width:280px;">
                     								{foreach item=arr from=$keyoptions}
 											{if $arr[0] eq $APP.LBL_NOT_ACCESSIBLE}
-                    									<option value="{$arr[0]}" {$arr[2]}>{$arr[0]}</option>
+                    									<option label="{$arr[0]}" value="{$arr[0]}" {$arr[2]}>{$arr[0]}</option>
```
Comment (joebordes) [inline Smarty/templates/DetailViewUI.tpl]: Please explain to me the reason for adding the label attribute here. I don't think it is needed for anything

**Comment by joebordes** [comment]
I manually merged these changes..
I had to do it manually for two reasons: one is that your base corebos is very different from the master branch, the other is because the PR overlaps with others you have already sent. The process is like this:

- fork coreBOS
- you keep your master branch in sync with coreBOS master brach
- for every change you want to incorporate you create a branch, no matter how smal
- you develop and test the feature on your branch
- you make a PR of the branch
- I evaluate and merge it into the master branch
- you pull the master branch changes into your forked master branch and you can eliminate the branch you created for developing the feature as the code is already in the master
- you create a new branch and develop something more and repeat

HTH, but don't hesitate to ask if it is not clear

File: Smarty/templates/DetailViewUI.tpl (hunk 1 above)
Comment (Luke1982) [inline Smarty/templates/DetailViewUI.tpl]: Honestly I don't remember exactly why I placed this. Let me remove it on develop install and see if it breaks anything.

**Comment by Luke1982** [comment]
Thanks,

Yes, I need to keep my fork more up to date. What I don't know is, if I merge my fork with the master branch, will the branches of my fork also be updated?

**Comment by joebordes** [comment]
No, branches are not modified
You would have to update each branch that you want, or start a new one.

**Comment by Luke1982** [comment]
Thanks for your help. I just updated my fork and will update my develop install accordingly to test and expand the theme.

**Pull request closed without merging**